.travis.yml

```diff
@@ -16,6 +16,9 @@ matrix:
   - go: 1.7
     env:
     - CROSSDOCK=true
+  - go: 1.7
+    env:
+    - DOCKER=true
```

Comment:
I have to admit that I don't know for sure what this does. I *guess* this will trigger different executors on Travis, but I just did a copy/paste of the sections above.